src/summary/profile/demographics-card.component.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { getCurrentPatient } from "@openmrs/esm-api";
+import SummaryCard from "../cards/summary-card.component";
+import { match } from "react-router";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import dayjs from "dayjs";
+
+export default function DemographicsCard(props: DemographicsCardProps) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient =>
+      setCurrentPatient(patient)
+    );
+
+    return () => subscription.unsubscribe();
+  });
+
+  return (
+    <SummaryCard name="Demographics" match={props.match}>
+      <SummaryCardRow linkTo="/">
+        <SummaryCardRowContent>
+          <VerticalLabelValue
+            label="Family"
+            value={currentPatient && currentPatient.name[0].family + ","}
+            valueStyles={{
+              textTransform: "uppercase"
+            }}
+          />
+          <VerticalLabelValue
+            label="Given"
+            value={currentPatient && currentPatient.name[0].given.join(" ")}
+          />
+        </SummaryCardRowContent>
```

Comment:
Huh, this design does make things kind of hard to configure, doesn't it. This code is a fine implementation of the design as it is. But I think it would be great to have the flexibility to format names, e.g., as they usually appear in Mexico, "Firstnames Lastnames", e.g. "José Emanuel Garcia López".

I'll take it up with Greg?

Reply:
👍 yeah let's ask greg about that

Reply:
If the answer is "Let's make it configurable," my suggestion is to continue forward with a concrete implementation and then refactor for configurability / extensibility once we have the features in place

Reply:
Sounds good to me

Reply:
Actually, we could configure this like

```
lastNameFirst: false,
familyNameStyle: "uppercase"
```

or something. Maybe the designs are fine.